src/Components/MediaCard/index.js

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import CardContent from "@mui/material/CardContent";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { Box } from "@mui/material";
+import styles from "./style.module.css";
+
+export default function MediaCard({
+  image,
+  title,
+  description,
+  click,
+  titleStyles,
+  descriptionStyles,
+  buttonStyles,
+}) {
+  return (
+    <Box>
+      <img src={`./img/${image}`} alt="1" />
+      <CardContent className={styles.text}>
+        <Typography
```

Comment:
You should have a loop and inside it you can add a Typography component instead of writing 3 Typography components. 
What if you had a lot of Typography components? How many lines will this file become?